src/types.ts

```diff
@@ -131,10 +149,13 @@ export interface Instance {
   /**
    * Finds all subdomains of a given domain
    * @param domainId (parent) domain id
-   * @param useDataStoreApi Optional, indicate whether to query with 
+   * @param useDataStoreApi Optional, indicate whether to query with
    * the DataStore or the Subgraph. Default is to use the DataStore
    */
-  getSubdomainsById(domainId: string, useDataStoreAPI?: boolean): Promise<Domain[]>;
+  getSubdomainsById(
+    domainId: string,
+    useDataStoreAPI?: boolean
```

Comment:
I know this isn't your code but we could fix it here. The code comment says the parameter is named `useDataStoreApi` but the parameter is actually named `useDataStoreAPI`.
They should be the same.

Reply:
Renamed